trojsten/submit/views.py

```diff
@@ -499,7 +501,30 @@ def task_submit_post(request, task_id, submit_type):
             return redirect(request.POST["redirect_to"])
         else:
             return redirect(reverse("task_submit_page", kwargs={"task_id": int(task_id)}))
-
+    elif submit_type == constants.SUBMIT_TYPE_TEXT:
+        submitted_text = "yes"  # TODO FIX it to get text from request
+        solution = task.text_submit_solution.lower()
+        submitted_text = unidecode(submitted_text.replace(" ", "").lower())
+        if submitted_text == solution:
+            if task.round.can_submit():
+                points = 6
+            elif timezone.now() < task.round.end_time + timedelta(days=4):
+                points = 4
+            elif timezone.now() < task.round.end_time + timedelta(days=7):
+                points = 2
+            else:
+                points = 0
+        else:
+            points = 0
```

Comment:
Ešte chýba odpočítanie bodov za nesprávne riešenia. Za každých 5 submitov, ktoré sú nesprávne (a boli vykonané pred prvým správnym, nech to neuberá body, ak submitujú aj po úspešnom vyriešní), sa odpočíta 1 bod.

Celkovo tie hinty ešte nevyzerajú byť implementované.

Reply:
Odpocitavanie neviem ako spravit.

Reply:
`points = max(points - wrong_submits / 5, 0)` ?

Reply:
ako ziskam `wrong_submits` ?